page_templates/style_guide.md

```diff
@@ -0,0 +1,73 @@
+
+# Jina AI Documentation Style Guide
+
+Guides are publicly available and represent the Jina AI brand and the quality of the product. Please adhere to the following steps to ensure basic professionalism in the docs.
+
+## Guiding principles:
+
+-   **Accessibility**. Everyone who needs to should understand our documents.
+
+-   **Professional**. A consistent writing and formatting style will give a professional look to the documents.
+
+## Specific guidelines:
+
+**[ For anything not covered here, we default to the [Google developer style guide.](https://developers.google.com/style)  ]**
+
+1.  Check the spelling and grammar in your contributions. Most editors include a spell checker or have an available spell-checking plugin. You can also paste your text into a Google Doc or other document software for a more robust spelling and grammar check. We use American English spelling.
+
+2.  Avoid run-on sentences. Generally, keep sentences to less than 13 words long.
+
+3.  When explaining technical steps, use a numbered step by step process. Don't list out every step in a single paragraph. For more details on this, see information [here](https://developers.google.com/tech-writing/one/lists-and-tables).
+
+4.  When explaining general concepts, paragraphs should be used. Yet paragraphs should not be over six sentences long.
+
+5.  Use the second person tense and describe actions to the user. (this is the same as active style or imperative style)
+
+> You should open Jina using python.
+> In your pod, you will find.
+
+6.  When using Jina terms, they should be Capitalized. Flow, Pod, JinaD. Anything found in the Jina glossary or Jina 101. Including Executor types (Evaluator, Segmenter, etc.)
+
+7. Don't capitalize Flow types (index, query). Don't capitalize the adjective or  `-tion`  form (i.e. evaluation, evaluating, segmenting, etc)
+
+8.  Do not use compound sentences, chains of clauses, and location-specific idioms, making text hard to understand and translate.
+
+9.  Avoid semicolons.
+
+10.  Don’t use abbreviations without explaining them first. With the expection of abbreviations and acronyms that are common in the computer industry. Examples include YAML, API, GIF, PDF, REST, ID. If you are unsure if you should define something or not, please ask in Slack.
+
+11. When discussing a Jina component inside a code context (i.e. referring to CLI output or actual Python/YAML code) use  Flow,  Pod, etc
```

Comment:
Use terms exactly as they are (or should be) portrayed in that context

Reply:
**Outside** of a code context, capitalize all elements (I think that's explained in a prior step, so can merge these points)

Reply:
Done